packages/react-vis/tests/plot/content-clip-path.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import ContentClipPath from '../../src/plot/content-clip-path';
```

Comment:
the component tests in the library usually involves creating a showcase example (which then also promotes the feature) and then basically verifying that it loads correctly. I'm willing to bend away from this pattern (i'm not sure what our story with story book is anymore), but i think it's a really two birds one stone to replace instances like your shallow rendering of the wrapper with a concrete example?

Reply:
There is a separate storybook that I created that allows them toggle the clipping on and off. 
I'm actually trying to break the dependency between the tests and the storybook as it requires some custom resolvers to ensure that the tests and and storybook point to the same version of react-vis.

Reply:
Does the storybook actually get published anywhere, or is it just for developers to play with the charts locally?

Reply:
I think it was meant to be published onto the website, but that that integration never got finished? Typically when i'm developing on react-vis I tend use the showcase. In fact I can't say if i've ever even opened the story book?

Reply:
If you are trying to keep storybook and tests seperate, this suggests that you should move that demo into the showcase?

Reply:
I tend to use the storybook as I'm developing as I find that there is too much going on (number of graphs) in the showcase. 
We eventually want to put more options and profiling tools inside of the storybook. 

 I'm actually working on another change to make development with the storybook even easier

Reply:
> If you are trying to keep storybook and tests seperate, this suggests that you should move that demo into the showcase?

I can put it in the showcase as well